miniapp/miniapp_recipes.cpp

```diff
@@ -80,35 +81,36 @@ class basic_cell_recipe: public recipe {
 
     cell_size_type num_cells() const override { return ncell_; }
 
-    cell get_cell(cell_gid_type i) const override {
+    util::unique_any get_cell(cell_gid_type i) const override {
         auto gen = std::mt19937(i); // TODO: replace this with hashing generator...
 
         auto cc = get_cell_count_info(i);
         const auto& morph = get_morphology(i);
         unsigned cell_segments = morph.components();
 
-        auto cell = make_basic_cell(morph, param_.num_compartments, cc.num_targets,
+        auto wrapped_cell = make_basic_cell(morph, param_.num_compartments, cc.num_targets,
                         param_.synapse_type, gen);
+        auto& c = util::any_cast<cell&>(wrapped_cell);
```

Comment:
Hmmmm, I have actually reverted this code! I don't know why it has showed up in your review.
I agree that `cell` is not the best name because it clashes with the type name.